src/as_execution/abi.rs

```diff
@@ -13,23 +13,31 @@ use wasmer::{AsStoreMut, AsStoreRef, FunctionEnvMut, Memory};
 use crate::env::{get_remaining_points, sub_remaining_gas_abi, ASEnv};
 use crate::settings;
 
-use super::common::{call_module, create_sc, function_exists};
+use super::common::{call_module, create_sc, function_exists, local_call};
 use super::error::{abi_bail, ABIResult};
-use super::local_call;
 
 macro_rules! get_memory {
     ($env:ident) => {
         match $env.get_ffi_env().memory.as_ref() {
             Some(mem) => mem,
-            _ => abi_bail!("No memory in env"),
+            _ => abi_bail!("AssemblyScript memory is missing from the environment"),
         }
     };
 }
 
+pub(crate) fn get_env(ctx: &FunctionEnvMut<ASEnv>) -> ABIResult<ASEnv> {
```

Comment:
I think it would be nice to add a bit of documentation to specify that a check is done and will fail if we are in instantiation.